core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -443,6 +443,21 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     assert(list2DiskGet.get.readMethod === DataReadMethod.Disk)
   }
 
+  test("block manager crash test") {
```

Comment:
Can you reference a JIRA in the test title or otherwise provide more description of what case is supposed to be tested here? This isn't a super descriptive name as it stands now.  Also, the test name probably doesn't need to include the word "test".